docs/UserGuide.md

```diff
@@ -79,7 +79,7 @@ Adds an elderly to ContactMate.
 
 Format: `add i/NRIC n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS c/CALL_FREQUENCY [t/TAG]…​`
 
-* The call frequency is in days and has to be a positive number less than or equal to 7.
+* The call frequency is measured in days and must be within 1 to 7 (i.e., within one week).
```

Comment:
i think the other groups point on this was that its probably better to specify `a positive integer` even though we already said that it is in days, just to be clear
```suggestion
* The call frequency is measured in days and must be a positive integer less than or equal to 7 (e.g. 1, 2, ..., 7)
```

Reply:
okk, cos i am thinking like the range from 1 to 7 is more specified and alr implied that it should be positive integer, so i just keep the part only